server/routes/river-and-sea-levels.js

```diff
@@ -39,7 +60,7 @@ module.exports = [{
         }
         return h.view(route, { model })
       }
-      if ((typeof place === 'undefined') || (!place.isUK || place.isScotlandOrNorthernIreland)) {
+      if ((typeof place === 'undefined') || (notinUk(place))) {
```

Comment:
So i think the flow in the route is not right, and does not marry up with the filter functionality the page provides.  The get page by rloiId needs adding to the flow that I added for station filtering.

So for example to getStations() we should add station
```
    // get base stations
    stations = await getStations(place, taCode, station)
```
On hitting the page, flow wise we'd only expect to have one of the place, taCode, rloiId, so that means in getStations we can add additional check for if rloiId/station then return floodService.getStationsByRadius(station.x,station.y)

That means the stations returned for a rloiId can then be put through the filterStations function

Reply:
removed and added to new flow